lib/ratchet/transformer.ex

```diff
@@ -35,7 +35,6 @@ defmodule Ratchet.Transformer do
     Enum.find_value attributes, :none, fn
       {"data-scope", scope} -> {:scope, scope}
       {"data-prop", prop} -> {:property, prop}
-      _ -> false
```

Comment:
This is needed to support arbitrary non data-* attribute names. It's probably worth adding such an example to the test suite to prove the point. I will do that! Thanks 💃